packages/sanity/src/_internal/cli/actions/schema/storeSchemasAction.ts

```diff
@@ -0,0 +1,136 @@
+import {readFileSync} from 'node:fs'
+import path, {join, resolve} from 'node:path'
+
+import {type CliCommandArguments, type CliCommandContext} from '@sanity/cli'
+import chalk from 'chalk'
+
+import {
+  type CreateManifest,
+  type ManifestSchemaType,
+  type ManifestWorkspaceFile,
+} from '../../../manifest/manifestTypes'
+import {MANIFEST_FILENAME} from '../manifest/extractManifestAction'
+import {SANITY_WORKSPACE_SCHEMA_ID} from './schemaListAction'
+
+export interface StoreManifestSchemasFlags {
+  'path'?: string
+  'workspace'?: string
+  'id-prefix'?: string
+  'schema-required'?: boolean
+  'verbose'?: boolean
+}
+
+export default async function storeSchemasAction(
+  args: CliCommandArguments<StoreManifestSchemasFlags>,
+  context: CliCommandContext,
+): Promise<Error | undefined> {
+  const flags = args.extOptions
+  if (typeof flags.path === 'boolean') throw new Error('Path is empty')
```

Comment:
Shouldnt these checks be `!flags.path || typeof flags.path !== 'string'`. I'd say that captures intent better.